microsoft/testsuites/core/storage.py

```diff
@@ -176,6 +176,51 @@ def verify_resource_disk_io(self, log: Logger, node: RemoteNode) -> None:
             "content read from file should be equal to content written to file",
         ).is_equal_to(original_text)
 
+    @TestCaseMetadata(
```

Comment:
Should this test case in Azure standard test suite?

Reply:
updated